.github/workflows/deploy-stage.yml

```diff
@@ -26,3 +28,36 @@ jobs:
           cf_org: hhs-acf-ohs-hses
           cf_space: ct-stage
           push_arguments: "--vars-file config/deployment/stage.yml --var rails_master_key=$RAILS_MASTER_KEY --var hses_auth_client_secret=$HSES_AUTH_CLIENT_SECRET"
+
+      - name: Check for changes to migrations
+        id: changed-files
+        uses: tj-actions/changed-files@v1.1.2
+        with:
+          files: db/migrate
+
+      - name: Run migration
+        if: steps.changed-files.outputs.any_changed == 'true'
+        uses: 18f/cg-deploy-action@main
+        with:
+          cf_username: ${{ secrets.CF_USERNAME }}
+          cf_password: ${{ secrets.CF_PASSWORD }}
+          cf_org: hhs-acf-ohs-hses
+          cf_space: ct-stage
+          full_command: |
+            cf run-task complaints-tracker-stage -c "bundle exec rake db:migrate"
+
+
+      - name: "wait a minute for task to run"
```

Comment:
Not an ideal solution, but `run-task` kicks off in the background and I don't think there's a good way to programmatically check for status updates.